src/include/migraphx/op/reduce_op.hpp

```diff
@@ -173,24 +202,56 @@ struct reduce_op : op_name<Derived>
             static_cast<const Derived&>(*this).output(batch_shape)(val);
     }
 
-    argument compute(const dyn_output& dyn_out, std::vector<argument> args) const
+    argument compute(const shape& computed_shape,
+                     const std::vector<int64_t>& reduce_axes,
+                     argument& data_arg) const
```

Comment:
Can we name this function something else so we're not getting into possible future trouble with how operation.hpp handles the compute function?

Reply:
Sure.